Cargo.lock

```diff
@@ -7578,7 +7628,7 @@ dependencies = [
  "itertools 0.11.0",
  "madsim-aws-sdk-s3",
  "madsim-tokio",
- "opendal",
+ "opendal 0.39.0",
```

Comment:
We may also bump this `opendal` in a separate PR.

Reply:
It's caused by icelake upgrade